debian/rules

```diff
@@ -4,7 +4,7 @@
 PIP_INDEX_URL ?= https://pypi.yelpcorp.com/simple
 
 DISTRIB_CODENAME := $(shell sed -n 's|DISTRIB_CODENAME=\(.*\)|\1|p' </etc/lsb-release)
-PYTHON_VERSION := 3.7
+PYTHON_VERSION := $(if $(filter-out jammy,$(DISTRIB_CODENAME)),3.7,3.8)
```

Comment:
imo, it’s probably better to stick to a single python version to reduce the cognitive overhead of having two different versions under which your code will run (and also, to minimize differences between the environment that paasta code runs under)

Reply:
@nemacysts  Looks like taking it back to Python3.7 introduces an error, this might be why the original focal PR moved this to 3.8

From make itest_jammy on this branch, with 3.8 replaced with 3.7 in the Dockerfile and debian/control
```
Unpacking paasta-tools (0.127.2-yelp1) ...
dpkg: dependency problems prevent configuration of paasta-tools:
 paasta-tools depends on base-files (<< 11ubuntu) | python3.8; however:
  Version of base-files on system is 12ubuntu4.1.
  Package python3.8 is not installed.
  ```

Reply:
Looks like vkhromov is not available in the near future, so we may not be able to ask them for help here. @nemacysts what do you think about shipping as-is, and deferring poking at debian/control at a later date? The hackathon ends tomorrow.